dashboards/ebs/ebs_snapshot_detail.sp

```diff
@@ -118,21 +118,29 @@ query "aws_ebs_snapshot_age" {
 
 query "aws_ebs_snapshot_relationships" {
   sql = <<-EOQ
-    with snapshot as (select arn,snapshot_id,volume_id,volume_size,title,account_id,region,kms_key_id from aws_ebs_snapshot where arn = $1)
+    with snapshot as 
+    (
+      select
+        arn,
+        snapshot_id,
+        volume_id,
+        volume_size,
+        title,
+        account_id,
+        region,
+        kms_key_id 
+      from
+        aws_ebs_snapshot 
+      where
+        arn = $1
+    )
     select
       null as from_id,
       null as to_id,
       snapshot_id as id,
       title as title,
       'aws_ebs_snapshot' as category,
-      jsonb_build_object(
-        'ID', snapshot.snapshot_id,
-        'ARN', snapshot.arn,
-        'Size', snapshot.volume_size,
-        'Account ID', snapshot.account_id,
-        'Region', snapshot.region,
-        'KMS Key ID', snapshot.kms_key_id
-      ) as properties
+      jsonb_build_object( 'ID', snapshot.snapshot_id, 'ARN', snapshot.arn, 'Size', snapshot.volume_size, 'Account ID', snapshot.account_id, 'Region', snapshot.region, 'KMS Key ID', snapshot.kms_key_id ) as properties 
```

Comment:
Does our SQL formatter compact this to one line? I think it does look better spread out over multiple lines.